locales/fr.json

```diff
@@ -132,25 +132,25 @@
   "prompt.choice_key_to_insert": "Choisissez une clé à insérer",
   "prompt.choice_key_to_open": "Choisissez une clé à ouvrir dans l'éditeur...",
   "prompt.choice_locale": "Choisir une locale",
-  "prompt.config_locales_auto_success": "Chemin de la locale défini automatiquement sur \"{0}\".",
+  "prompt.config_locales_auto_success": "Chemin de la locale défini automatiquement sur «{0}».",
   "prompt.config_locales_button": "Configurer maintenant",
   "prompt.config_locales_info": "Configurer le répertoire de la locale pour votre projet.",
   "prompt.config_locales_success": "Chemin de la locale configurer avec succès.",
   "prompt.create_new_path": "Créer un nouveau chemin",
   "prompt.deepl_api_key_required": "La clé d'API DeepL est requise",
   "prompt.deepl_error_get_usage": "Erreur lors de l'obtention des données d'usage",
   "prompt.deepl_usage": "Utilisation de l'API DeepL APı {0} sur {1} caractères traduites",
-  "prompt.delete_key": "Êtes-vous sûr de supprimer la clé \"{0}\" de toutes les locales?\n\nCette opération ne peut PAS être annulée.",
-  "prompt.delete_keys_not_in_use": "Souhaitez-vous réellement supprimer {0} Clé(s) n'étant pas utilisée(s) ?",
+  "prompt.delete_key": "Êtes-vous sûr de supprimer la clé «{0}» de toutes les locales?\n\nCette opération ne peut PAS être annulée.",
+  "prompt.delete_keys_not_in_use": "Souhaitez-vous réèllement supprimer {0} Clé(s) n'étant pas utilisée(s) ?",
   "prompt.donate": "Donation",
-  "prompt.edit_key_in_locale": "{1} | Édition de la Clé \"{0}\"",
-  "prompt.enter_file_path_to_store_key": "Entrer le chemin du fichier où stocker la Clé \"{0}\"",
-  "prompt.enter_key_path": "Entrer le chemin de la clé pour \"{0}\"",
+  "prompt.edit_key_in_locale": "{1} | Edition de la Clé «{0}»",
+  "prompt.enter_file_path_to_store_key": "Entrer le chemin du fichier où stocker la Clé «{0}»",
+  "prompt.enter_key_path": "Entrer le chemin de la clé pour «{0}»",
   "prompt.enter_new_keypath": "Entrer le nouveau chemin des Clés",
   "prompt.error_on_parse_custom_regex": "Échec de l'analyse de la chaîne d'expression régulière {0}. Vérifiez votre configuration.",
   "prompt.existing_translation": "Traduction existante",
   "prompt.extraction_canceled": "Extraction annulée.",
-  "prompt.failed_to_locate_key": "Échec de la localisation de la Clé \"{0}\".",
+  "prompt.failed_to_locate_key": "Echec de la localisation de la Clé «{0}».",
```

Comment:
`Échec` to `Echec`: is it on purpose?

Reply:
same as previous.

Reply:
In French, putting accents on capital letters isn't considered as incorrect, both are valid translations. I found the transition to `«»` interesting btw, I don't feel like it was necessary but why not I guess